src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -55,6 +55,19 @@ disconnected](#get-disconnected-from-a-bluetooth-device).
 
 <img style="width:723px; max-height:250px" src="/web/updates/images/2015-07-22-interact-with-ble-devices-on-the-web/web-bluetooth-flag.png" alt="Web Bluetooth Flag highlighted in chrome://flags"/>
 
+### Available for Origin Trials
+
+In order to keep from prematurely baking the Web Bluetooth API design in before
```

Comment:
I might say this more positively: "In order to get as much feedback as possible from developers using the Web Bluetooth API in the field, we're also adding this feature in Chrome 53 as an origin trial for Chrome OS, Android M & Mac. During the origin trial, the API may still change in backward-incompatible ways before we freeze it into the web platform."